packages/module/package.json

```diff
@@ -24,9 +29,6 @@
   "release": {
     "branches": "main"
   },
```

Comment:
You'll need to delete this key for the release config file to be picked up by semantic release

Reply:
Done